wearApp/proguard-rules.pro

```diff
@@ -5,16 +5,8 @@
 -dontwarn kotlinx.serialization.KSerializer
 -dontwarn kotlinx.serialization.Serializable
 
-# Referenced by okhttp
```

Comment:
This shouldn't be needed with OkHttp 5 (or 4.11)

Maybe for the shaded version?

Reply:
OkHttp was 4.9 when [this was introduced](https://github.com/joreilly/Confetti/commit/e057483e146518d98352209407333a52824b3c4b), let me try to remove this.

Reply:
Looks like that works well : https://github.com/joreilly/Confetti/pull/712